pkg/api/api.go

```diff
@@ -315,10 +315,6 @@ func (hs *HTTPServer) registerRoutes() {
 			apiRoute.Group("/search-v2", hs.SearchV2HTTPService.RegisterHTTPRoutes)
 		}
 
-		if hs.Features.IsEnabledGlobally(featuremgmt.FlagUnifiedStorageSearch) {
-			apiRoute.Group("/unified-search", hs.UnifiedSearchHTTPService.RegisterHTTPRoutes)
```

Comment:
same thing available with https://github.com/grafana/grafana/pull/96907 -- BUT we can run this as a multi-tenant service 🎉